api/src/services/relations.ts

```diff
@@ -143,6 +143,15 @@ export class RelationsService {
 			);
 		}
 
+		const columnInfo = await this.schemaInspector.columnInfo(relation.collection, relation.field);
```

Comment:
Lets pull this from `this.schema` instead, to prevent the performance overhead of reading this info fresh from the database.

Reply:
Yes that's an unnecessary overhead. Thank you!